### PR TITLE
license: move from GPLv2 to AGPLv3 (base commit)

### PR DESCRIPTION
**Pull request by iGormilhit**
* NEW Changes the LICENSE file content.
* NEW Changes the files headers.
* Orders authors alphabetically.

Co-Authored-by: Igor Milhit <igor.milhit@rero.ch>

:warning: This commit is big, because it's the first commit of this work, and has been done before the solution of a specific branch. The next commits will be lighter.

### CODE REVIEW

File: rero_ils/static/scss/rero_ils/styles.scss [hunk 1]
```diff
@@ -1,11 +1,20 @@
...
- * Copyright (C) 2018 RERO.
- *
- * RERO ILS is free software; you can redistribute it and/or modify it
- * under the terms of the GNU General Public License;
- * See license file for more details.
+
+  RERO ILS
+  Copyright (C) 2019 RERO
+  Copyright (C) 2015-2018 CERN
+  This program is free software: you can redistribute it and/or modify
```
Comment (Garfield-fr) [inline rero_ils/static/scss/rero_ils/styles.scss]: Blank line after copyright

File: rero_ils/static/scss/rero_ils/variables.scss [hunk 2]
```diff
@@ -1,12 +1,21 @@
...
- *
- * RERO ILS is free software; you can redistribute it and/or modify it
- * under the terms of the GNU General Public License;
- * See license file for more details.
- */
+
+  RERO ILS
+  Copyright (C) 2019 RERO
+  Copyright (C) 2015-2018 CERN
+  This program is free software: you can redistribute it and/or modify
```
Comment (Garfield-fr) [inline rero_ils/static/scss/rero_ils/variables.scss]: Blank line after copyright

File: rero_ils/static/templates/rero_ils/count.html [hunk 3]
```diff
@@ -1,10 +1,21 @@
 <!--
-  This file is part of RERO ILS.
+
+  RERO ILS.
   Copyright (C) 2016-2018 CERN.
   Copyright (C) 2019 RERO.
```
Comment (Garfield-fr) [inline rero_ils/static/templates/rero_ils/count.html]: Remove dot after ILS and copyright text

File: rero_ils/static/templates/rero_ils/facets.html [hunk 4]
```diff
@@ -1,10 +1,21 @@
 <!--
-  This file is part of RERO ILS.
+
+  RERO ILS.
   Copyright (C) 2016-2018 CERN.
   Copyright (C) 2019 RERO.
```
Comment (Garfield-fr) [inline rero_ils/static/templates/rero_ils/facets.html]: Remove dot after ILS and copyright text

File: rero_ils/static/templates/rero_ils/pagination.html [hunk 5]
```diff
@@ -1,10 +1,21 @@
 <!--
-  This file is part of RERO ILS.
+
+  RERO ILS.
   Copyright (C) 2016-2018 CERN.
   Copyright (C) 2019 RERO.
```
Comment (Garfield-fr) [inline rero_ils/static/templates/rero_ils/pagination.html]: Remove dot after ILS and copyright text

File: rero_ils/static/templates/rero_ils/searchbar.html [hunk 6]
```diff
@@ -1,10 +1,21 @@
 <!--
-  This file is part of RERO ILS.
+
+  RERO ILS.
   Copyright (C) 2016-2018 CERN.
   Copyright (C) 2019 RERO.
```
Comment (Garfield-fr) [inline rero_ils/static/templates/rero_ils/searchbar.html]: Remove dot after ILS and copyright text

File: rero_ils/templates/rero_ils/_editor_button_actions.html [hunk 7]
```diff
@@ -1,3 +1,21 @@
+{# -*- coding: utf-8 -*-
+
+  RERO ILS.
+  Copyright (C) 2019 RERO.
```
Comment (Garfield-fr) [inline rero_ils/templates/rero_ils/_editor_button_actions.html]: Remove dot after ILS and copyright text

File: rero_ils/templates/rero_ils/footer.html [hunk 8]
```diff
@@ -1,3 +1,21 @@
+{# -*- coding: utf-8 -*-
+
+  RERO ILS.
+  Copyright (C) 2019 RERO.
```
Comment (Garfield-fr) [inline rero_ils/templates/rero_ils/footer.html]: Remove dot after ILS and copyright text

File: rero_ils/templates/rero_ils/forgot_password.html [hunk 9]
```diff
@@ -1,11 +1,21 @@
 {# -*- coding: utf-8 -*-
 
-  This file is part of RERO ILS.
+  RERO ILS.
+  Copyright (C) 2019 RERO.
```
Comment (Garfield-fr) [inline rero_ils/templates/rero_ils/forgot_password.html]: Remove dot after ILS and copyright text

File: rero_ils/templates/rero_ils/frontpage.html [hunk 10]
```diff
@@ -1,3 +1,22 @@
+{# -*- coding: utf-8 -*-
+
+  RERO ILS.
+  Copyright (C) 2019 RERO.
+  Copyright (C) 2015-2018 CERN.
```
Comment (Garfield-fr) [inline rero_ils/templates/rero_ils/frontpage.html]: Remove dot after ILS and copyright text

File: rero_ils/templates/rero_ils/head.html [hunk 11]
```diff
@@ -1,10 +1,19 @@
 {# -*- coding: utf-8 -*-
 
-  This file is part of RERO ILS.
-  Copyright (C) 2018 RERO.
+  RERO ILS.
+  Copyright (C) 2019 RERO.
```
Comment (Garfield-fr) [inline rero_ils/templates/rero_ils/head.html]: Remove dot after ILS and copyright text

File: rero_ils/templates/rero_ils/header.html [hunk 12]
```diff
@@ -1,11 +1,20 @@
 {# -*- coding: utf-8 -*-
 
-  This file is part of RERO ILS.
-  Copyright (C) 2018 RERO.
+  RERO ILS.
+  Copyright (C) 2019 RERO.
```
Comment (Garfield-fr) [inline rero_ils/templates/rero_ils/header.html]: Remove dot after ILS and copyright text

File: rero_ils/templates/rero_ils/header_frontpage.html [hunk 13]
```diff
@@ -1,3 +1,21 @@
+{# -*- coding: utf-8 -*-
+
+  RERO ILS.
+  Copyright (C) 2019 RERO.
```
Comment (Garfield-fr) [inline rero_ils/templates/rero_ils/header_frontpage.html]: Remove dot after ILS and copyright text

File: rero_ils/templates/rero_ils/header_login.html [hunk 14]
```diff
@@ -1,3 +1,21 @@
+{# -*- coding: utf-8 -*-
+
+  RERO ILS.
+  Copyright (C) 2019 RERO.
```
Comment (Garfield-fr) [inline rero_ils/templates/rero_ils/header_login.html]: Remove dot after ILS and copyright text

File: rero_ils/templates/rero_ils/javascript.html [hunk 15]
```diff
@@ -1,10 +1,20 @@
 {# -*- coding: utf-8 -*-
 
-  This file is part of RERO ILS.
-  Copyright (C) 2018 RERO.
+  RERO ILS.
+  Copyright (C) 2019 RERO.
```
Comment (Garfield-fr) [inline rero_ils/templates/rero_ils/javascript.html]: Remove dot after ILS and copyright text

File: rero_ils/templates/rero_ils/login_user.html [hunk 16]
```diff
@@ -1,11 +1,20 @@
 {# -*- coding: utf-8 -*-
 
-  This file is part of RERO ILS.
-  Copyright (C) 2015-2018 CERN.
-  Copyright (C) 2018 RERO.
+  RERO ILS.
+  Copyright (C) 2019 RERO.
```
Comment (Garfield-fr) [inline rero_ils/templates/rero_ils/login_user.html]: Remove dot after ILS and copyright text

File: rero_ils/templates/rero_ils/macros/header_menus.html [hunk 17]
```diff
@@ -1,3 +1,22 @@
+{# -*- coding: utf-8 -*-
+
+  RERO ILS.
+  Copyright (C) 2019 RERO.
```
Comment (Garfield-fr) [inline rero_ils/templates/rero_ils/macros/header_menus.html]: Remove dot after ILS and copyright text

File: rero_ils/templates/rero_ils/macros/macro.html [hunk 18]
```diff
@@ -1,10 +1,19 @@
 {# -*- coding: utf-8 -*-
 
-  This file is part of RERO ILS.
-  Copyright (C) 2018-2019 RERO.
+  RERO ILS.
+  Copyright (C) 2019 RERO.
```
Comment (Garfield-fr) [inline rero_ils/templates/rero_ils/macros/macro.html]: Remove dot after ILS and copyright text

File: rero_ils/templates/rero_ils/macros/messages.html [hunk 19]
```diff
@@ -1,11 +1,19 @@
 {# -*- coding: utf-8 -*-
 
-  This file is part of RERO ILS.
-  Copyright (C) 2015-2018 CERN.
+  RERO ILS.
   Copyright (C) 2019 RERO.
```
Comment (Garfield-fr) [inline rero_ils/templates/rero_ils/macros/messages.html]: Remove dot after ILS and copyright text

File: rero_ils/templates/rero_ils/page.html [hunk 20]
```diff
@@ -1,10 +1,19 @@
 {# -*- coding: utf-8 -*-
 
-  This file is part of RERO ILS.
-  Copyright (C) 2018 RERO.
+  RERO ILS.
+  Copyright (C) 2019 RERO.
```
Comment (Garfield-fr) [inline rero_ils/templates/rero_ils/page.html]: Remove dot after ILS and copyright text

File: rero_ils/templates/rero_ils/page_admin.html [hunk 21]
```diff
@@ -1,13 +1,21 @@
 {# -*- coding: utf-8 -*-
 
-  This file is part of RERO ILS.
-  Copyright (C) 2018-2019 RERO.
+  RERO ILS.
+  Copyright (C) 2019 RERO.
```
Comment (Garfield-fr) [inline rero_ils/templates/rero_ils/page_admin.html]: Remove dot after ILS and copyright text

File: rero_ils/templates/rero_ils/page_cover.html [hunk 22]
```diff
@@ -1,13 +1,22 @@
 {# -*- coding: utf-8 -*-
 
-  This file is part of RERO ILS.
+  RERO ILS.
+  Copyright (C) 2019 RERO.
```
Comment (Garfield-fr) [inline rero_ils/templates/rero_ils/page_cover.html]: Remove dot after ILS and copyright text

File: rero_ils/templates/rero_ils/page_error.html [hunk 23]
```diff
@@ -1,11 +1,21 @@
 {# -*- coding: utf-8 -*-
 
-  This file is part of RERO ILS.
+  RERO ILS.
+  Copyright (C) 2019 RERO.
```
Comment (Garfield-fr) [inline rero_ils/templates/rero_ils/page_error.html]: Remove dot after ILS and copyright text

File: rero_ils/templates/rero_ils/register_user.html [hunk 24]
```diff
@@ -1,11 +1,21 @@
 {# -*- coding: utf-8 -*-
 
-  This file is part of RERO ILS.
+  RERO ILS.
+  Copyright (C) 2019 RERO.
```
Comment (Garfield-fr) [inline rero_ils/templates/rero_ils/register_user.html]: Remove dot after ILS and copyright text

File: rero_ils/templates/rero_ils/reset_password.html [hunk 25]
```diff
@@ -1,11 +1,21 @@
 {# -*- coding: utf-8 -*-
 
-  This file is part of RERO ILS.
+  RERO ILS.
+  Copyright (C) 2019 RERO.
```
Comment (Garfield-fr) [inline rero_ils/templates/rero_ils/reset_password.html]: Remove dot after ILS and copyright text

File: rero_ils/templates/rero_ils/search.html [hunk 26]
```diff
@@ -1,3 +1,22 @@
+{# -*- coding: utf-8 -*-
+
+  RERO ILS.
+  Copyright (C) 2019 RERO.
+  Copyright (C) 2015-2018 CERN.
```
Comment (Garfield-fr) [inline rero_ils/templates/rero_ils/search.html]: Remove dot after ILS and copyright text

File: rero_ils/templates/rero_ils/search_header.html [hunk 27]
```diff
@@ -1,13 +1,22 @@
 {# -*- coding: utf-8 -*-
 
-  This file is part of Invenio.
+  RERO ILS.
+  Copyright (C) 2019 RERO.
```
Comment (Garfield-fr) [inline rero_ils/templates/rero_ils/search_header.html]: Remove dot after ILS and copyright text

File: rero_ils/templates/rero_ils/sidebar.html [hunk 28]
```diff
@@ -1,3 +1,22 @@
+{# -*- coding: utf-8 -*-
+
+  RERO ILS.
+  Copyright (C) 2019 RERO.
+  Copyright (C) 2015-2018 CERN.
```
Comment (Garfield-fr) [inline rero_ils/templates/rero_ils/sidebar.html]: Remove dot after ILS and copyright text

File: rero_ils/templates/rero_ils/tombstone.html [hunk 29]
```diff
@@ -1,11 +1,21 @@
 {# -*- coding: utf-8 -*-
 
-  This file is part of RERO ILS.
+  RERO ILS.
+  Copyright (C) 2019 RERO.
```
Comment (Garfield-fr) [inline rero_ils/templates/rero_ils/tombstone.html]: Remove dot after ILS and copyright text

File: rero_ils/templates/rero_ils/trackingcode.html [hunk 30]
```diff
@@ -1,3 +1,21 @@
+<!--
+
+  RERO ILS.
+  Copyright (C) 2019 RERO.
```
Comment (Garfield-fr) [inline rero_ils/templates/rero_ils/trackingcode.html]: Remove dot after ILS and copyright text

File: rero_ils/static/js/rero_ils/rero_ils.js [hunk 31]
```diff
@@ -1,14 +1,23 @@
...
- * Copyright (C) 2017-2018 RERO.
- *
- * Invenio is free software; you can redistribute it and/or modify it
- * under the terms of the GNU General Public License as published by
- * the Free Software Foundation; either version 2 of the License,
- * or (at your option) any later version.
- */
+
+RERO ILS
+Copyright (C) 2016-2018 CERN.
```
Comment (Garfield-fr) [inline rero_ils/static/js/rero_ils/rero_ils.js]: Remove dot

File: rero_ils/static/js/rero_ils/rero_ils.js [hunk 32]
```diff
@@ -1,14 +1,23 @@
...
+Copyright (C) 2016 TIND.
```
Comment (Garfield-fr) [inline rero_ils/static/js/rero_ils/rero_ils.js]: Remove dot

File: AUTHORS.rst [hunk 33]
```diff
@@ -1,35 +1,28 @@
...
   * Aly Badr
-  * Gianni Pante
   * Nicolas Labat
+  * Johnny Mariéthoz
+  * Igor Milhit
   * Miguel Moreira
+  * Gianni Pante
   * Nicolas Prongué
+  * Peter Weber
+  * Bertrand Zuchuat
```
Comment (BadrAly) [inline AUTHORS.rst]: Add Sébastien?

Comment (iGormilhit) [reply AUTHORS.rst]: You're right. I'll add @sebastiendeleze in a future commit.

**Pull request merged**